src/main/java/it/pagopa/ecommerce/commons/utils/NpgApiKeyConfiguration.java

```diff
@@ -0,0 +1,199 @@
+package it.pagopa.ecommerce.commons.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.vavr.control.Either;
+import it.pagopa.ecommerce.commons.client.NpgClient;
+import it.pagopa.ecommerce.commons.exceptions.NpgApiKeyConfigurationException;
+
+import java.util.*;
+import java.util.function.Function;
+
+/**
+ * This class handles all NPG api keys such as default one and per methods psp.
+ * Use {@link NpgApiKeyConfigurationBuilder} class to initialize a new class
+ * instance api keys
+ */
+public class NpgApiKeyConfiguration {
+
+    /**
+     * NpgApiKeyConfiguration builder class
+     */
+    public static class NpgApiKeyConfigurationBuilder {
```

Comment:
What about calling this just `Builder`?

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-commons/pull/188/commits/54e4f2f55e9aaa2b18a0421d06c609252205ac16